package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "lint": "eslint . --fix",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "lint:check": "eslint .",
```

Comment:
i don't think you need to worry about backwards compatibility
we are in pre-alpha
breaking changes should be expected
just don't forget to change the the command in the workflow

Reply:
I just figured I'd be nice to people who are already used to `lint:check`. I don't think it hurts to keep it in there

Reply:
lint:check isn't for people to use 
its just for the workflow
if the workflow had the fix flag it would not warn auto-fixable problems

Reply:
I'd say lint:check is definitely for people to use as well. I use it when changing ESLint stuff so that I can see how the entire codebase is affected. And some people might prefer to run ESLint manually instead of being bothered by constant linting errors in their editor

Reply:
1 - when you create a electron-vite project the default "lint" script has the fix flag
2 - i created the "lint:check" script only thinking in the workflow use for it
3 - if you want to make this change i think at least we should not have 2 scripts that do the same thing 
but that's it 
do what you think is best
i don't want to keep arguing about this

Reply:
I personally just did not expect that when I ran `npm run lint`, which to me reads as a read-only action ("lint my files and give me the result"), that it would then also modify my files. This may simply depend on what someone is used to, but you can use e.g. Sourcegraph to see that the vast majority of public projects don't use `--fix` in the regular `lint` script, so I think that matches user expectations better.

Also sorry if this came off as arguing, I'm just explaining my thought process really

Reply:
> Also sorry if this came off as arguing, I'm just explaining my thought process really

don't worry
I didn't know that "arguing" had a "bad" meaning
the equivalent word in Portuguese "argumentando" is more neutral
like "creating arguments", i suppose this sounds better?
maybe "reasoning" would be a better word? 
idk

Reply:
Ah yea I get what you mean lol all good :+1: